Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -799,7 +799,7 @@ extension AlertManager {
     }
     
     func presentCouldNotResetLoopAlert(error: Error) {
-        let alert = UIAlertController(title: "Could Not Restart Tidepool Loop", message: "While trying to restart Tidepool Loop an error occured.\n\n\(error.localizedDescription)", preferredStyle: .alert)
+        let alert = UIAlertController(title: "Could Not Restart Loop", message: "While trying to restart Loop an error occured.\n\n\(error.localizedDescription)", preferredStyle: .alert)
```

Comment:
appName?

Reply:
This is non user facing, for Tidepool Loop, so I thought Loop might work here. But I can update it to string formatting, too.  This is something that would've been nice to catch on the initial review when "Tidepool Loop" landed in shared code.